.github/workflows/ci.yml

```diff
@@ -498,3 +498,21 @@ jobs:
 
       - name: Update Homebrew Formula
         run: ./update-formula.sh ${{needs.draft_release.outputs.create_release_name }}
+
+  ci_build_all:
```

Comment:
WE need to reuse existing job and settings.

Reply:
working on it

Reply:
made the changes, please review @tusharmath